reana_server/rest/workflows.py

```diff
@@ -417,8 +420,17 @@ def create_workflow(user):  # noqa
             workflow = _get_workflow_with_uuid_or_name(
                 response["workflow_id"], str(user.id_)
             )
+
+            # FIXME: this is a quick fix, should be refactored when possible
```

Comment:
We don't need this FIXME comment; please rather say "# This is necessary for GitLab integration" and then create a new issue on GitHub for further refactoring of the whole shebang.